src/core/Form/TextInput/TextInput.md

```diff
@@ -29,3 +29,25 @@ import { TextInput } from 'suomifi-ui-components';
   />
 </>;
 ```
+
+```js
+import { TextInput } from 'suomifi-ui-components';
+
+const [errorState, setErrorState] = React.useState(false);
+const statusText = errorState
+  ? 'You entered invalid data'
+  : undefined;
+const variant = errorState ? 'error' : 'default';
+
+<>
+  <TextInput
+    labelText="Test TextInput"
+    statusText={statusText}
+    variant={variant}
+  />
+
+  <button onClick={() => setErrorState(!errorState)}>
+    Toggle error state
+  </button>
+</>;
+```
```

Comment:
I added an example to styleguidist how to use the `statusText` and `variant`.  Do others see this as useful or should it be removed?

I also used the basic button for not to have this example dependent of Button component. Is this okay, for others too?